networks/__init__.py

```diff
@@ -0,0 +1,12 @@
+import torch.nn as nn
+
+from networks.caml import CAML
+from networks.kim_cnn import KimCNN
+from networks.xml_cnn import XMLCNN
+
+
+def get_weight_init_func(config):
+    def init_weights(m):
```

Comment:
we also have --weight-init. For consistency maybe we should always use "init_weights"?

Reply:
Use "init_weight" in all places.